### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lpg-tech**

### CODE REVIEW

File: input/ch.fhir.ig.ch-lab-order.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <title value="CH LAB-Order (R4)" />
   <status value="active" />
   <experimental value="false" />
-  <date value="2020-09-03" /> <!-- last changed -->
+  <date value="2021-06-14" /> <!-- last changed -->
```
Comment (isels1) [inline input/ch.fhir.ig.ch-lab-order.xml:17]: 15. because of merge?

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,5 @@ resources/implementationguide/ig-*.*
 org.hl7.*.jar
 publisher.jar
 publisher.jar.bak
-ch.fhir.ig.ch-orf.xml
\ No newline at end of file
+ch.fhir.ig.ch-orf.xml
```
Comment (isels1) [inline .gitignore:10]: why ignoring this file?

**Review by isels1 — APPROVED** [review]
Approved with one note: why is the "ch.fhir.ig.ch-orf.xml" file in git ignore? Should be one of the core files to get the build done...?

File: .gitignore (hunk 2 above)
Comment (lpg-tech) [reply .gitignore:10]: I don't remember why this file is in gitignore. Anyway, it does no harm, because it is not used. I will remove it.

**Pull request merged**